fq-warmup-lib/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'java'
+}
+
+compileJava   {
+    sourceCompatibility = '1.8'
+    targetCompatibility = '1.8'
+}
+
+group 'eu.xenit.ass'
+
+repositories {
+    mavenCentral()
+    jcenter()
```

Comment:
Do we need `jcenter()` for some artifact?
If not, better to not add it at all.

Reply:
yes, it's needed for some transitive dependency. I added a comment